app/build.gradle

```diff
@@ -1,91 +1,54 @@
-repositories {
-    mavenCentral()
-    google()
-    maven { url "https://jitpack.io" }
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-kapt'
 }
 
-apply plugin: "com.android.application"
-apply plugin: "kotlin-android"
-apply plugin: "kotlin-kapt"
-
-//Android
 android {
-    compileSdkVersion android_target_sdk
-    buildToolsVersion android_build_tools_version
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
 
     defaultConfig {
-        applicationId "com.masmovil.mini"
-        minSdkVersion 21
-        targetSdkVersion android_target_sdk
-        versionCode android_version_code
-        versionName android_version_name
-        multiDexEnabled true
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-    }
+        applicationId "com.example.androidsample"
+        minSdkVersion 16
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
 
-    buildFeatures {
-        viewBinding = true
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
-
     compileOptions {
-        sourceCompatibility "1.8"
-        targetCompatibility "1.8"
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
     }
-
     kotlinOptions {
-        jvmTarget = "1.8"
-    }
-
-    lintOptions {
-        abortOnError false
-    }
-
-    configurations.all {
-        //This is library is included with two different versions
-        resolutionStrategy.force "com.google.code.findbugs:jsr305:3.0.1"
-    }
-
-    packagingOptions {
-        exclude "META-INF/rxjava.properties"
+        jvmTarget = '1.8'
     }
 }
 
 dependencies {
-    implementation fileTree(dir: "libs", include: ["*.jar"])
-
-    implementation project(":masmini-android")
-    implementation project(":masmini-rx2")
-    implementation project(":masmini-rx2-android")
-    implementation project(":masmini-kodein-android")
```

Comment:
I'd keep these imports so all modules get compiled when compiling the app (except the RX ones)